doc/content/verification_and_validation/ver-1kb.md

```diff
@@ -0,0 +1,83 @@
+# ver-1kb
+
+# Henry’s Law Boundaries with No Volumetric Source
+
+## General Case Description
+
+Two enclosures are separated by a membrane that allows diffusion according to Henry’s law, with no volumetric source present. Enclosure 2 has twice the volume of Enclosure 1.
+
+## Case Set Up
+
+This verification problem is taken from [!cite](ambrosek2008verification).
+
+Over time, the pressures of T$_2$, which diffuses across the membrane in accordance with Henry’s law, will gradually equilibrate between the two enclosures.
+
+The diffusion process in each of the two enclosures can be described by the following equations:
+
+\begin{equation}
+\frac{\partial C_1}{\partial t} = - \nabla D \nabla C_1
+\end{equation}
+
+\begin{equation}
+\frac{\partial C_2}{\partial t} = - \nabla D \nabla C_2
+\end{equation}
+
+where $C_1$, $C_2$ represent the concentration fields in enclosures 1 and 2 respectively, and $D$ denotes the diffusivity.
+
+The concentration in Enclosure 1 is related to the partial pressure and concentration in Enclosure 2 via the interface sorption law:
+
+\begin{equation}
+C_1 = K P_2^n = K \left( C_2 RT \right)^n
+\end{equation}
+
+where $R$ is the ideal gas constant in J/mol/K, $T$ is the temperature in K, $K$ is the solubility, and $n$ is the exponent of the sorption law. For Henry’s law, $n=1$.
+
+## Results
```

Comment:
In this section, please add a note (see other cases with notes and warnings about typos) saying that we do not get the same kinetics as the results shown in TMAP7, which we attribute to a different diffusivity value.